src/services/storage/index.ts

```diff
@@ -28,7 +30,13 @@ export class Storage implements StorageDriver {
 	}
 	public async initialize(): Promise<void> {
 		// Waits for 1 minute (6 * 10,000ms) before failing
-		await runWithRetries(this._driver.initialize.bind(this._driver), { retries: 6, retryTime: 10 * 1000 })
+		try {
+			logger.debug(`Running Storage.initialize`)
+			await runWithRetries(this._driver.initialize.bind(this._driver), { retries: 6, retryTime: 10 * 1000 })
+		} catch (err) {
+			logger.error(`Storage.initialize failed: ${err}`)
+			throw new GeneralError('Could not initialize (Storage.initialize)')
```

Comment:
Can you throw the actual error here? This function is not propagated back to the client, so it's safe to throw it
```suggestion
			throw err
```